python/paddle/tensor/stat.py

```diff
@@ -253,6 +253,140 @@ def numel(x, name=None):
     return out
 
 
+def nanmedian(x, axis=None, ignore_nan=True, keepdim=True, name=None):
+    """
+    Compute the median along the specified axis, while ignoring NaNs.
+
+    Args:
+        x (Tensor): The input Tensor, it's data type can be int32, int64, float16, float32, float64.
+        axis (None|int|list|tuple, optional):
+            The axis along which to perform median calculations ``axis`` should be int.
+            ``axis`` should be in range [-D, D), where D is the dimensions of ``x`` .
+            If ``axis`` is less than 0, it works the same way as :math:`axis + D`.
+            If ``axis`` is None, median is calculated over all elements of ``x``. Default is None.
+        ignore_nan (bool, optional): Whether to ignore nan values when median was calculated.
+            If `ignore_nan` is True, the calculation process is the same as `median` operator.
```

Comment:
If `ignore_nan` is False, the calculation process is the same as `median` operator?

Reply:
modified.

Reply:
> please add link of RFC(API design documents) in description

already added.